lynx/common/actions/chop.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+
+from lynx.common.actions.action import Action
+from lynx.common.vector import Vector
+from lynx.common.actions.removeObject import RemoveObject
+from lynx.common.actions.createObject import CreateObject
+from lynx.common.object import Object
+from lynx.common.actions.common_requirements import CommonRequirements
+
+@dataclass
+class Chop(Action):
+	"""
+	Simple action used to hit/destroy object, which stands on the given destination.
+	"""
+	object_id: int = -1

```

Comment:
What do you think about creating a static class with variables such as `INACTIVE_OBJECT_ID = -1`,  because in the future it will be easier to understand what -1 means.

Reply:
Added a TODO